build/windows/test.ps1

```diff
@@ -16,7 +16,7 @@ if (-not $?)
 
 Write-Output "--- Running tests"
 
-go test $workspace\pkg\... $workspace\cmd\... $workspace\internal\... $workspace\test\...
+go test -ldflags '-X github.com/newrelic/infrastructure-agent/internal/integrations/v4/integration.minimumIntegrationIntervalOverride=2s' $workspace\pkg\... $workspace\cmd\... $workspace\internal\... $workspace\test\...
```

Comment:
I was wondering if we can run them separately from unit tests. (using a tag like we do for harvest tests)

Reply:
Surely I miss something but I don't really understand the point of running the config protocol tests separately...  If I'm not wrong the harvest tests are only executed on Linux but we need to run the config protocol tests on both Windows and *nix,  so if we do the suggested change, then we should run `go test ....`. twice in this script., right?

Reply:
As discussed internally, we're going to leave them in the standard test suite, and possibly move them to the slow tests once this is done: https://github.com/newrelic/infrastructure-agent/issues/502